package/base-files/files/sbin/sysupgrade

```diff
@@ -282,7 +284,7 @@ fi
 
 if [ -n "$CONF_BACKUP" ]; then
 	do_save_conffiles "$CONF_BACKUP"
-	exit $?
```

Comment:
I had to lookup the explanation for that one, you could have added to the commit msg:
"remove redundant $? from exit builtin call", or something like that.